web-client/public/js/graph.js

```diff
@@ -1070,7 +1071,10 @@ var drawGraph = function (nodes, links, positiveWeights, negativeWeights, sheetT
 
             // For whatever reason the arc collapses to a point if the beginning
             // and ending points of the arc are the same, so kludge it.
-                        x2 = d.source.x + d.source.textWidth / 1.2;
+                        if(d.source.textWidth > 135){
+                            nodeShift += 0.07;
+                        }
+                        x2 = d.source.x + d.source.textWidth / 1.2 * nodeShift;
```

Comment:
Can the values `135`, `.0.07`, and `1.2` be given names? We’re in tech debt with regard to magic numbers so it would be nice if we can avoid adding more…

Reply:
You are right. I assigned names for each values and changed the code accordingly. I will look out for magic numbers in the future!